evap/contributor/templates/contributor_evaluation_form.html

```diff
@@ -26,7 +26,7 @@ <h3>
         {{ evaluation.full_name }} ({{ evaluation.course.semester.name }})
```

Comment:
- The search text entered in a multi-select field should be removed after adding an entry